waspc/data/Generator/templates/server/src/auth/providers/email/login.ts

```diff
@@ -0,0 +1,20 @@
+import { verifyPassword } from "../../../core/auth.js";
+import { handleRejection } from "../../../utils.js";
+import { findUserBy, createAuthToken } from "../../utils.js";
+
+export const login = handleRejection(async (req, res) => {
+    const args = req.body || {};
+    const user = await findUserBy<'email'>({ email: args.email });
```

Comment:
Don't we also need to make sure they are verified?

Reply:
There are two options we considered:
1. Unverified users have access by default and the developer decides within their actions what to do
2. Unverified users should not be allowed to log in at all

The first flow sounds to me like it could be better with providing extra onboarding steps while the user didn't verify their e-mail. I'm not sure which one we want to go with. Maybe we put it as an option in the Wasp file?